lightning/src/ln/channelmanager.rs

```diff
@@ -229,6 +229,22 @@ pub enum PendingHTLCRouting {
 		/// [`ReleaseHeldHtlc`] onion message.
 		hold_htlc: Option<()>,
 	},
+	/// A dummy HTLC hop which does not represent a real routing decision.
+	///
+	/// Dummy HTLCs are introduced to pad a blinded path. When such an HTLC is received, the dummy
```

Comment:
We already use "pad" to refer to making all hops the same size. Let's re-word this to avoid overloading that word.